app/app-controllers/doubleIconsController.js

```diff
@@ -3,41 +3,40 @@ import Astro from 'astro/astro-full';
 import BackboneEvents from 'vendor/backbone-events';
 import DoubleIconsPlugin from '../app-plugins/doubleIconsPlugin';
 
-var DoubleIconsController = function(headerId, generateLeftIcon, generateRightIcon) {
+const DoubleIconsController = function(headerId, generateLeftIcon, generateRightIcon) {
     this.headerId = headerId;
     this.generateLeftIcon = generateLeftIcon;
     this.generateRightIcon = generateRightIcon;
 };
 
-var _setLeftIcon = function(doubleIcons, address) {
+const _setLeftIcon = function(doubleIcons, address) {
     doubleIcons.setLeftIcon(address);
```

Comment:
Talked to Jake... we weren't waiting on the promise returned by `setLeftIcon` before so no need to `await` here.